packages/contracts/contracts/UnirepApp.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.0;
 import {Unirep} from '@unirep/contracts/Unirep.sol';
 import {EpochKeyVerifierHelper} from '@unirep/contracts/verifierHelpers/EpochKeyVerifierHelper.sol';
-
+import {EpochKeyLiteVerifierHelper} from '@unirep/contracts/verifierHelpers/EpochKeyLiteVerifierHelper.sol';
```

Comment:
Could you briefly explain newly imported LiteVerifier in PR description and also link to doc for future reference?

Reply:
Should I add a doc for smart contract in our repo? (I will add this as an issue, first create a empty file for now and add the document in the future.)

Reply:
> Should I add a doc for smart contract in our repo? (I will add this as an issue, first create a empty file for now and add the document in the future.)

IMO you can reference with just a link to unirep package.